packages/core/src/navigation-item/NavigationItem.tsx

```diff
@@ -75,34 +130,69 @@ export const NavigationItem = forwardRef<HTMLDivElement, NavigationItemProps>(
       "--saltNavigationItem-level": `${level}`,
     };
 
+    const isParent = parent || href === undefined;
+    const elementProps = {
+      className: clsx(
+        withBaseName("wrapper"),
+        {
+          [withBaseName("active")]: active || blurActive,
+          [withBaseName("blurActive")]: blurActive,
+          [withBaseName("rootItem")]: level === 0,
+        },
+        withBaseName(orientation)
+      ),
+      children: (
```

Comment:
This sort of breaks things for me, this common children has isParent in it but the parentProps and linkProps are split by parent, seems a bit weird but not a deal breaker.